.cspell/custom-dictionary.txt

```diff
@@ -0,0 +1,533 @@
+n_speci
+objs
+octupole
+oned
+orcid
+outcoming
+orientable
+overvoltage
+parabolical
+pdbx
+PEEM
+phaseplate
+piezo
+plottable
+pmatrix
+polariton
+polfilter
+polycapillary
+polylines
+polynom
+polytope
+polytopes
+PRNG
+proxigram
+pseudosymmetry
+ptychography
+PYCCAPT
+Qdev
+Qmean
+RCSB
+RDMS
+RHIT
+RRNG
+Raman
+raylight
+rdeform
+readback
+recalibrations
+rediscretization
+rediscretized
+reflectometer
+reflectron
+regrid
+retardance
+Ronchigram
+SAS
+sasdata
+sasdetector
+sasentry
+sasinstrument
+sasnote
+sasprocess
+sasprocessnote
+sassample
+sassource
+sastransmission
+saveframe
+SAXS
+scanpoints
+semiaxes
+SERS
+sextett
+shapepar
+slowaxisname
+SNSnxtranslate
+soller_angle
+spatiotemporally
+spatstat
+spectr.
+spectralfilter
+sphx
+SPLEED
+SPIE
+SPPERS
+spwidth
+stepsize
+subprocessing
+supercontinuum
+supermirror
+supermirrors
+stigmators
+STXM
+SXES
+TCSPC
+TERMS
+TERS
+Tdev
+tessellator
+tetrahedralization
+tetrahedralize
+thermalized
+traj
+transfermatrices
+turbomolecular
+twotheta
+twotime
+underconstrained
+unidata
+unvalidated
+vibropolishing
+VLEED
+voxelization
+watertightness
+wavelen
+wavelenghts
+WAXS
+WSAS
+WWPDB
+xaxis
+xdim
+XFEL
+XPCS
+ydim
+yaxis
+zaxis
+zener
+Zincblende
+zoneplate
+
+# Names used for publications, places, companies, etc.
+Acta.Crysta
+Adv. Opt. Techn.
+Alaukik
+Ankerest
+Aspnes
+Azzam
+Bashara
+Berkels
+Bonse
+Borkowski
+Breen
+Bünger
+Cameca
+cameca
+Cameca's
+Cardona
+DESY
+Diehl
+Dierksen
+Dillamore
+Dimkou
+Dresselhaus
+Doniach-Sunjic
+EIKOS
+Erlangen
+Felfer
+Fleischmann
+Fluerasu
+fRecoVolume
+Freysoldt
+Frigo
+Gehrke
+geiser
+Gernot
+Gernot
+Ginkel
+Glan-Focault
+Gottstein
+Graeff
+gatan_imaging
+Götz
+Güntherodt
+Hawkes
+Hoshen-Kopelman
+Heusler
+icatproject
+inchi
+Inspico
+invizo
+IUPAC
+j.matchar.2016.04.008
+J. Appl. Cryst.
+J. Phys. D: Appl. Phys.
+Jägle
+Jorio
+Katnagallu
+Keerthi
+Kikuchi
+Kitrick
+Kocks
+Kopelman
+Kuehbach
+Kühbach
+Krakauer
+Krakauer/Seidman
+LAWATAP
+Leoben
+Levenberg-Marquardt
+Lewiner
+Lorensen
+Manz
+MatWerk
+Markus
+McStas
+Mersenne
+Monajem
+Morawiec
+Nellist
+NCNR
+NFDI
+NION
+Nion
+nion
+Normarski
+OMERO
+Ophus
+ORCID
+orientation_rodrigues
+ORNL
+Pedersen
+Pennycock
+Popko
+Poschenrieder
+Protochips
+Preiß
+Primig
+Rainer
+Reichmann
+Reimer
+Remmele
+Rielli
+ripleyk
+Romaner
+Rollett
+rollett_holm
+rotation_rodrigues
+Rowenhorst
+RRAW
+Saxena
+schottky
+sebald_gottstein
+Seblock
+seblock
+Seidman
+Senarmont
+shermann
+Shermann
+TAPHR
+Techn
+Thilo
+Tougaard
+Typke
+von Neumann
+VORONOI
+Voronoi
+voronoi
+Wellenreuther
+Weppelman
+Windl
+Zener
+Zhou
+Zollner
+
+# Software
+bluesky
+ifes_apt_tc_data_modeling
+nanochem
+paraprobe
+paraprobe
+paraview
+parmsetup
+pyiron
+pynxtools
+XDMF
+
+# other languages
+Institut
+
+# en-UK which requires to many changes
+analyse
+analyser
+analysers
+Neighbour
+neighbour
+normalisation
+normalise
+normalised
+oxidising
+polarisation
+practise
+presense
+summarised
+synchronised
+utilised
+visualisation
+
+# misc
+aequatorial
```

Comment:
Just change the instance to the modern spell

Reply:
Thats a tricky one aequatorial is used in NXsas* type definitions, which would demand for a vote and invalidate some NeXus files, recommend to make an issue for modernizing these and instead leaving it "aequatorial" for now here

Reply:
No problem - we can live with that choice